pmultiqc/modules/mzidentml/mzidentml.py

```
@@ -0,0 +1,1569 @@
+
+    @staticmethod
+    def dis_decoy(protein_name):
+        if config.kwargs["decoy_affix"] not in protein_name:
+            return "TARGET"
+        elif protein_name.split(";") == 1:
+            return "DECOY"
+        else:
+            if config.kwargs["affix_type"] == "prefix":
+                if list(
+                    filter(
+                        lambda x: lambda x: not x.startswith(config.kwargs["decoy_affix"]),
```

Ruff (0.13.1)

Unused lambda argument: `x`

(ARG005)